### PR TITLE
Update Spatial Query and Layer Ordering examples

### PR DESCRIPTION
**Pull request by pmacMaps**
Thanks to @jwasilgeo for a good replacement URL for Portland Neighborhoods.

### CODE REVIEW

File: src/pages/examples/spatial-queries.hbs [hunk 1]
```diff
@@ -97,36 +95,40 @@ layout: example.hbs
...
 
     // query the service executing the selected relation with the selected input geometry
     neighborhoods.query()[relationship.value](inputGeometry).ids(function(error, ids){
-      previousIds = ids;
-      for (var i = ids.length - 1; i >= 0; i--) {
-        neighborhoods.setFeatureStyle(ids[i], { color: 'red', weight: 2 });
-      };
+      if (error) {
+        console.log('Error with query: ' + error);
+      } else {
```
Comment (jwasilgeo) [inline src/pages/examples/spatial-queries.hbs]: to account for when `ids` can be `null` instead of an array
```suggestion
      } else if (ids) {
```

Comment (pmacMaps) [reply src/pages/examples/spatial-queries.hbs]: I've made updates.  Please review again.  I sometimes have trouble with removing trailing spaces, so hopefully that is resolved before.

And yeah, the errors with the null ids were present before.  But I think your suggestion resolves that issue as well.

File: src/pages/examples/spatial-queries.hbs [hunk 2]
```diff
@@ -97,36 +95,41 @@ layout: example.hbs
...
 
   // reset all features back to their regularly defined styles
-  function reset(){
+  function reset() {
     for (var i = previousIds.length - 1; i >= 0; i--) {
       neighborhoods.resetStyle(previousIds[i]);
     };
   }
-
+
```
Comment (jwasilgeo) [inline src/pages/examples/spatial-queries.hbs]:
```suggestion

```

File: src/pages/examples/spatial-queries.hbs [hunk 3]
```diff
@@ -97,36 +95,41 @@ layout: example.hbs
...
       neighborhoods.resetStyle(previousIds[i]);
     };
   }
-
+  
   // query the API and highlight features
   function query(){
     reset();
-
+
```
Comment (jwasilgeo) [inline src/pages/examples/spatial-queries.hbs]:
```suggestion

```

**Comment by jwasilgeo** [comment]
Related to #229 

Thanks a ton, @pmacMaps!

**Pull request merged**